lang/tl.json

```
@@ -0,0 +1,96 @@
+{
+  "select_language": "Piliin ang iyong wika:",
+  "select_language_subtitle": "Pwede rin ibago ang iyong wika sa settings mamaya!",
+  "english": "Tagalog",
+  "continue": "Tumuloy",
+  "welcome_title": "Maligayang Pagdating sa Fire!",
+  "welcome_message": "Isang tool para sa depensa ng komunidad ng mga migrante sa Los Angeles.",
+  "a_chirla_project": "Project ng CHIRLA",
+  "must_be_over_13": "Dapat ikaw ay igit sa edad na 13 na gumamit ng Fire.",
+  "verify_over_13": "Pinapatunayan mo na ikaw ay igit sa edad na 13 sa pagpindot ng \"tumuloy\".",
+  "protect_yourself": "Protektahan ang iyong sarili.",
+  "protect_yourself_subtitle": "Magkaroon ng mabilis na access sa isang libre na hotline para sa mga emergency na kailangan mo sila i-contact sakaling na detenihin ka o ang mga mahal mo sa buhay ng ICE.",
+  "know_your_rights": "Alamin ang iyong mga karapatan.",
+  "know_your_rights_subtitle": "Alamin kung ano ang pwede mong gawin pag nasa labas ng bahay mo ang ahente ng ICE at sa iba pang mga sitwasyon.",
+  "connect_with_organizations_subtitle": "Maghanap ng mga organisasyon na malapit sayo na nagbibigay ng mga libre o abot-kayang ligal na serbisyo",
+  "select_hotline": "Pumili ng isang emergency hotline",
+  "select_hotline_subtitle": "Piliin ang isang emergency hotline na pinakamalapit sa kung saan ka gumugugol ng pinakamadaming oras.",
+  "what_is_hotline": "Ano ang emergency hotline?",
+  "what_is_hotline_content": "Ang mga emergency hotline ay nagsisilbi ng iba't ibang lugar ng Southern California kung sakali ay ikaw o ang mga mahal mo sa buhay ay na detenhin ng mga awtoridad ng imigrasyon. ",
```

```suggestion
  "what_is_hotline_content": "Ang mga emergency hotline ay nagsisilbi ng iba't ibang lugar ng Southern California kung sakali ay ikaw o ang mga mahal mo sa buhay ay na detenhin ng mga awtoridad ng imigrasyon.",
```

example of trailing space we could clean up in script